packages/core/src/A.js

```diff
@@ -19,4 +19,8 @@ A.propTypes = {
   href: PropTypes.string.isRequired,
 };
 
+A.defaultProps = {
+  href: ''
```

Comment:
`href` is required & hence won't need default prop

Reply:
@kilemensi What happens when we don't supply a `href` value?

Reply:
You should get an error @nerdlet ... otherwise why use this component if you don't have a link?

Reply:
Got it...In that case, I am gonna just leave it as is